packages/cozy-client/src/__tests__/withMutations.spec.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
```

Comment:
I am not sure. All tests in Cozy-Client are under the `/packages/*/src/__tests__/` folders.

Reply:
Not all tests are under the `__tests__`, there's actually a mixture :

```
./src/helpers.spec.js
./src/associations/helpers.spec.js
./src/associations/HasMany.spec.js
./src/associations/HasOne.spec.js
./src/associations/HasManyFiles.spec.js
./src/associations/HasManyInPlace.spec.js
./src/CozyLink.spec.js
./src/queries/optimize.spec.js
./src/queries/dsl.spec.js
./src/Query.spec.jsx
./src/CozyClient.spec.js
./src/hoc.spec.js
./src/Schema.spec.js
./src/store/documents.spec.js
./src/store/queries.spec.js
./src/__tests__/connect.spec.jsx
./src/__tests__/Provider.spec.jsx
./src/__tests__/StackLink.spec.js
./src/__tests__/associations.spec.js
./src/__tests__/withMutation.spec.jsx
./src/__tests__/ObservableQuery.spec.js
./src/__tests__/store.spec.js
```

I prefer the spec next to the file way as it is easier to check the spec along the file and you do not have to recreate the folder structure, it is also easier when you are moving file since you do not have to remember to move the fixture.

Reply:
Ok for me. The flow we recenlty identified with @CPatchane on Cozy-Harvest-Lib is that test file are also copyied in the `dist` folder during build. In Cozy-Harvest-Lib, we are using the `--copy-files` options to copy the locales files into the `dist` folder.

Reply:
Done 👌